requirements/openvino.txt

```diff
@@ -1,6 +1,6 @@
 defusedxml==0.7.1
 requests==2.26.0
 networkx~=2.5
-nncf@ git+https://github.com/openvinotoolkit/nncf@37a830a412e60ec2fd2d84d7f00e2524e5f62777#egg=nncf
+nncf==2.1.0
```

Comment:
It is because we cannot upload anomalib to PyPI if the dependencies are git links. This is also the reason why we include freia stuff inside anomalib

Reply:
Ah, yes of course. In this case should we request a review from Alexander? I think this specific commit was added to the requirements in his recent PR, so there might be a reason why we don't just use the pypi package.

Reply:
Fair point

Reply:
NNCF 2.1.0 is no support features for anomalib, but we are planning release 2.2.0 in 1-2 weeks.